src/GSvar/FilterWidget.cpp

```diff
@@ -55,26 +55,6 @@ FilterWidget::FilterWidget(QWidget *parent)
 	ui_.clearn_btn->setMenu(new QMenu());
 	ui_.clearn_btn->menu()->addAction("Clear filters", this, SLOT(clearFilters()));
 	ui_.clearn_btn->menu()->addAction("Clear filters and ROI", this, SLOT(clearFiltersAndRoi()));
-
-	try
-	{
-		loadTargetRegions();
-	}
-	catch(Exception& e)
-	{
-		Log::warn("Target region data for filter widget could not be loaded from NGSD: " + e.message());
-	}
-
-	try
```

Comment:
Filters have nothing to do the with NGSD, or?

Reply:
GSvar loads "ROIs of NGSD processing systems"